prusti/tests/pass/parse/after_expiry.rs

```diff
@@ -0,0 +1,22 @@
+// compile-flags: -Zprint-desugared-specs -Zprint-typeckd-specs -Zskip-verify -Zhide-uuids
+// normalize-stdout-test: "[a-z0-9]{32}" -> "$(NUM_UUID)"
+// normalize-stdout-test: "[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}" -> "$(UUID)"
+
+#![feature(register_tool)]
+#![register_tool(prusti)]
+
+use prusti_contracts::*;
+
+#[after_expiry_if(a => a, a)]
```

Comment:
Could you add a test that contains a Rust match expression and check that it is parsed correctly?